src/video_canister_package/src/index.ts

```diff
@@ -0,0 +1,243 @@
+import { Identity } from "@dfinity/agent";
+import { Principal } from "@dfinity/principal";
+
+import { getVideoCanisterActor, getWalletCanisterActor, managementPrincipal } from "./common";
+import { MetaInfo } from "./canisters/video_canister/video_canister.did";
+import { IDL } from "@dfinity/candid";
+
+export interface Video{
+  "name": string,
+  "description": string,
+  "version": bigint,
+  "owner": Principal,
+  "videoBuffer": Buffer,
+}
+
+export interface CreationVideo{
+  "name": string,
+  "description": string,
+  "videoBuffer": Buffer,
+}
+
+const spawnPrincipal = Principal.fromText("ryjl3-tyaaa-aaaaa-aaaba-cai"); 
+
+const creationCycles: bigint = BigInt(200_000_000_000);
+
+const chunkSize = 1024;
```

Comment:
Eventuell könnten wir allgemein die Konstanten aus common.ts und index.ts in ein eigenes File packen und importieren? Wäre mmn übersichtlicher. open for discussion

Reply:
resolved 1c674a82f5826adc89c2880c7bd8fa636d2e689a